prisma/schema.prisma

```diff
@@ -14,6 +14,18 @@ model Videos {
   youtubeid String
   suggested Boolean  @default(false)
 
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now())
+}
+
+model Blogposts {
+  id String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  title    String
+  description String
+  url String
+  suggested Boolean  @default(false)
```

Comment:
Mulig du sa dette, men hvordan er det tiltenkt at `suggested`-feltet skal fungere?

Reply:
Om vi anbefaler artikkelen. Må kanskje utvide denne modellen litt med flere sånne felter!

Reply:
Men vil vi ikke egentlig anbefale alle? 😝 eller hvordan skal dette kalkuleres?

Reply:
Ah nei! Vi må manuelt sette feltet til `true` hvis vi skal anbefale artikkelen. Alle blir satt default til false 😄

Reply:
Åja, det har jeg egentlig ikke tenkt så mye over... Sånn som det er nå, så er det bare at hvis vi selv ønsker å fremheve noen bloggposter, så setter vi denne til true. Burde nok tenkes litt mer rundt dette! Hvis vi skal gå for RSS-feed, så kan vi jo egentlig bare droppe hele den modellen og gå rett mot tags.